src/prompt.c

```diff
@@ -149,7 +147,7 @@ char	*set_prompt(t_list *env_list)
 	char	*directory_tmp;
 	char	*directory;
 
-	username = get_username(env_list);
+	username = ft_strjoin(getenv("USER"), "@");
```

Comment:
Username part of the prompt now relies solely on USER env variable being set before running minishell. If we used the one from our env copy, it would change to whatever we set there (e.g. ahoj) and that's not how bash work.